npm-publish/bin/publish.sh

```diff
@@ -6,7 +6,7 @@ set -o nounset   # error on undefined vars
 set -o pipefail  # error if piped command fails
 
 # Default variables
-RELEASE_BRANCH=`echo $PR_HEAD_REF | awk  -F '--' '{print $2}' | awk -F '--' '{print $1}'`
+RELEASE_BRANCH=$(echo "$PR_HEAD_REF" | awk  -F '--' '{print $2}' | awk -F '--' '{print $1}')
```

Comment:
[SC2006](https://www.shellcheck.net/wiki/SC2006): Use $(...) notation instead of legacy backticked \`...\`.

Reply:
[SC2086](https://www.shellcheck.net/wiki/SC2086): Double quote to prevent globbing and word splitting.